docs/en/query_language/dicts/external_dicts_dict_sources.md

```diff
@@ -233,6 +278,19 @@ The dictionary configuration in ClickHouse:
 </yandex>
 ```
 
+or
+
+```sql
+CREATE DICTIONARY table_name (
+    id UInt64,
+    some_column UInt64 DEFAULT 0
+)
+PRIMARY KEY id
+SOURCE(ODBC(connection_string 'DSN=myconnection' table 'postgresql_table'))
```

Comment:
some extra ident might make this look better

Reply:
Just two parameters... looks OK for me.